### DISCUSSION TITLE
Please update error for blog demo.

### DISCUSSION THREAD

**Discussion by jaclover** [remix-run/remix#1119] [Q&A] [answered]
There are a few typos in the docs. 
https://remix.run/docs/en/v1/tutorials/blog
 -
`export type Post = {
  slug: string;
  title: string;
};`
should read: 
```
export type Post = {
  slug: string,
  title: string,
};
```
This is wrong is many places.
and also:
`export type PostMarkdownAttributes = {
  title: string;
};`
should read:
`export type PostMarkdownAttributes = {
  title: string
};`

**Comment by machour** [comment, marked as answer]
both `,` and `;` are supported when declaring a type.
